### ISSUE TITLE
 kk suffix not used by hybrid/scaled

### ISSUE THREAD

**Issue by Leimeroth** [ICAMS/calphy#50]
Using command line args it should be possible to pass accelerator suffixes to all pair styles without having to modify the input scripts themselves. This seems to work corretly for the pace pair style, which uses my GPU as it is supposed to.
The hybrid/scaled style however throws an error at me at some point of free energy calculations:
```
Exception: ERROR: Must use pair_style hybrid/scaled/kk with Kokkos (src/src/pair_hybrid_scaled.cpp:242)
Kokkos::Cuda ERROR: Failed to call Kokkos::Cuda::finalize()
```
I am not sure whether this is somehow related to the inputs to lammps that are used by calphy or a lammps bug since I don't see why -sf kk should not apply to hybrid_scaled automatically

**Comment by srmnitc** [comment]
I think we need to modify calphy to use `hybrid/scaled/kk` instead of simply `hybrid/scaled`. The [docs](https://docs.lammps.org/pair_hybrid.html) seem to indicate this. I will take a look today. 

The easiest way would be to add a flag to calphy input and change pair style depending on if the flag is present. Does this sound like a decent solution?

**Comment by Leimeroth** [comment]
I modified my fork to directly use `hybrid/scaled/kk` and lammps does not recognize it:
```
ERROR: Unrecognized pair style 'hybrid/scaled/kk' (src/src/force.cpp:280)
Last command: pair_style       hybrid/scaled/kk v_one pace product v_fscale pace product
```
Since the error before told me to use it I guess the error is on the lammps side in this case

**Comment by srmnitc** [comment]
Yes, I assume so. As long as the library version is compiled with `kk` option, it should work I assume.

**Comment by srmnitc** [comment]
Issue related to LAMMPS. https://github.com/lammps/lammps/issues/3492
